performancetest/src/main/java/org/corfudb/performancetest/LogUnitPerformanceTest.java

```diff
@@ -0,0 +1,138 @@
+package org.corfudb.performancetest;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import org.corfudb.protocols.wireprotocol.DataType;
+import org.corfudb.protocols.wireprotocol.LogData;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.clients.LogUnitClient;
+import org.corfudb.util.serializer.Serializers;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.*;
+
+/**
+ * Created by Nan Zhang and Lin Dong on 10/23/19.
+ */
+
+public class LogUnitPerformanceTest {
+    private String endPoint = "localhost:9000";
+    private int metricsPort = 1000;
+    private long seed = 1024;
+    private int randomBoundary = 100;
+    private Random random;
```

Comment:
These fields should read value from properties.

Reply:
Good point